pyproject.toml

```diff
@@ -39,7 +39,7 @@ test = ["pytest>=6.0", "pytest-cov", "pytest-qt"]
 pyqt5 = ["PyQt5"]
 pyside2 = ["PySide2"]
 pyqt6 = ["PyQt6"]
-pyside6 = ["PySide6!=6.4.0,!=6.4.0.1"]
+pyside6 = ["PySide6!=6.4.0,!=6.4.0.1,!=6.4.1"]
```

Comment:
nope, undo... this is what I was saying in https://github.com/pymmcore-plus/pymmcore-widgets/pull/70#issuecomment-1325302514

this is actually a bug in our code (it's _super_ important not to just exclude various versions just to get tests passing unless you know exactly why it's failing.  since it makes it harder for the package to exist in the ecosystem  without dependency conflicts)